assets/shaders/viewport_render.wgsl

```diff
@@ -292,7 +292,7 @@ fn get_node_occupancy_bitmap(sized_node_meta: u32) -> u32 {
 }
 
 // Functionality-wise this function is more generic, than its coutnerpart
```

Comment:
May I suggest a: `coutnerpart` -> `counterpart`. I am not sure How I can add a "suggested change"

Reply:
Sure! :) It's the button on the top left by the way: 
![image](https://github.com/davids91/shocovox/assets/4211067/645a5431-332c-43a3-937a-005277fb2598)